docs/user/tutorials/01-40-20-aws-redis-instance.md

```diff
@@ -61,13 +71,17 @@ To instantiate Redis and connect the Pod with only the required fields, use the
    ```bash
    redis-cli -h $HOST -p $PORT --tls PING
    ```
+  
    If your setup was successful, you get `PONG` back from the server.
 
 ## Advanced Setup
 
-To specify advanced features (such as Redis version, configuration, and maintenance policy) and set up auth and TLS, use the following setup:
+To specify advanced features, such as Redis version, configuration, and maintenance policy, and set up auth and TLS, use the following setup:
```

Comment:
```suggestion
To specify advanced features, such as Redis version, configuration, and maintenance policy, and set up auth, use the following setup:
```
TLS is now always enabled implicitly.
The tutorial was out of date.
Please add this update as part of your refactor :)